fastapi_jsonapi/atomic/schemas.py

```diff
@@ -151,7 +150,16 @@ def _validate_one_of_ref_or_href(cls, values: dict):
         # TODO: pydantic V2
         raise ValueError(msg)
 
-    @root_validator
+    @classmethod
+    def _get_value_from_dict_or_obj(cls, obj: Any, key: str):
+        if hasattr(obj, key):
+            return getattr(obj, key)
+        elif isinstance(obj, dict) and key in obj:
```

Comment:
we don't need `elif` when previous `if` returns result. please change this `elif` to `if`